src/wodin.ts

```diff
@@ -0,0 +1,30 @@
+import * as base from "./base";
+import type { OdinModelConstructable, Solution } from "./model";
+import {runModel} from "./model";
+import type { UserType } from "./user";
+
+// tslint:disable-next-line:variable-name
+export function wodinRun(Model: OdinModelConstructable, pars: UserType,
```

Comment:
The sensitivity and fits will be exported here in some form yes - but they'll be slightly different I expect because they might not be simple functions.

What do you mean "additional functions from the same endpoint" though? If you mean listed here, then yes!

Reply:
Maybe I'm misunderstanding/have forgotten. At the moment we get the runner from odin.api via an endpoint, and the payload of that endpoint is the single runner function. I thought we were going to continue that pattern, with the odin.api using this package to return the runner method. So was asking how the current runner endpoint will change/be augmented to return the model fit and sensitivity functions too. We could pull in this package directly in wodin front end, but I thought we'd decided to centralise that access through odin.api.

Reply:
We can have odin.api ship the webpacked module (forgive my terminology) so it will be an object with run/fit/sensitivity in there, yeah. Ideally we can configure webpack to create a version that does not contain anything that is not needed too (requiring multiple webpack targets  - I presume that's easy enough to do)